versions/1/manifest.spec.yml

```diff
@@ -114,6 +114,7 @@ spec:
           src:
             description: Relative path to the icon's image file.
             type: string
+            format: relative-path
```

Comment:
This should be documented in `CONTRIBUTING.md` for anyone who wants to add elements to the spec in the future that could use `relative-path` validation.

Reply:
Fixed